src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,126 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
use `GenerationType.SEQUENCE` for postgre, in postgre `GenerationType.AUTO` doesn't generate a sequence for each table

Reply:
I would use GenerationType.IDENTITY, but the difference is minimal
http://www.ngdeveloper.com/generationtype-identity-vs-generationtype-sequence-vs-generationtype-auto/

Reply:
perfect!! 👌